src/tokenize/Tokenizer.ts

```diff
@@ -20,9 +39,22 @@ export class Tokenizer {
             if (tok.type === ERRORTOKEN && isSpace(tok.string)) {
                 continue;
             }
+            // if we're in fmode we may need to adjust the lineno and offset
+            if (this._fmode) {
+                this._adjust_offset(tok);
+            }
```

Comment:
I wasn't sure whether to do something cleverer here. Like change the _push function if we're in fmode to avoid this check for ever push. Maybe it's not important.

Reply:
this is fine with me 😄